baseapp/baseapp_test.go

```diff
@@ -510,15 +510,20 @@ func TestValidatorChange(t *testing.T) {
 
 	// Assert that validator updates are correct.
 	for _, val := range valSet {
+
+		pk, err := tmtypes.PB2TM.PubKey(val.PubKey)
 		// Sanity
-		assert.NotEqual(t, len(val.PubKey), 0)
+		assert.Nil(t, err)
 
 		// Find matching update and splice it out.
 		for j := 0; j < len(valUpdates); {
```

Comment:
Add `j++` to the for loop here no need to have it at the end of the loop

Reply:
Updated.